libraries/api-extractor-model/src/model/ApiInterface.ts

```diff
@@ -69,7 +71,7 @@ export interface IApiInterfaceJson
  * @public
  */
 export class ApiInterface extends ApiItemContainerMixin(
-  ApiNameMixin(ApiTypeParameterListMixin(ApiReleaseTagMixin(ApiDeclaredItem)))
+  ApiNameMixin(ApiTypeParameterListMixin(ApiReleaseTagMixin(ApiReadonlyMixin(ApiDeclaredItem))))
```

Comment:
How can an interface be readonly? I think you maybe meant to add this to `ApiPropertySignature` instead?

Reply:
Yes, when I was reading through the different pieces measured I read through ApiPropertySignature and mistakenly added functionality to interfaces post-fact. Addressed